packages/material-ui/src/RangeWidget/RangeWidget.tsx

```diff
@@ -19,7 +19,7 @@ const RangeWidget = ({
   const sliderProps = { value, label, id, name: id, ...rangeSpec(schema) };
 
   const _onChange = (_: any, value?: number | number[]) => {
-    onChange(value ? options.emptyValue : value);
+    onChange(value ? value : options.emptyValue);
```

Comment:
It it likely that the same code is in `packages/mui/src/RangeWidget/RangeWidget.tsx`. Please update that as well

Reply:
Good point, done.